ssl/statem/statem_lib.c

```diff
@@ -1559,10 +1558,9 @@ static int is_tls13_capable(const SSL *s)
          * more restrictive so check that our sig algs are consistent with this
          * EC cert. See section 4.2.3 of RFC8446.
          */
-        eckey = EVP_PKEY_get0_EC_KEY(s->cert->pkeys[SSL_PKEY_ECC].privatekey);
-        if (eckey == NULL)
-            continue;
-        curve = EC_GROUP_get_curve_name(EC_KEY_get0_group(eckey));
+        curve = EVP_PKEY_get_EC_KEY_curve_nid(s->cert->pkeys[SSL_PKEY_ECC]
```

Comment:
Shouldn't this be `evp_pkey_get_EC_KEY_curve_nid` (i.e. in lowercase)?

Reply:
Yes.  There are a few of these, I've already corrected them locally